tests/testing_nurbs.py

```diff
@@ -315,6 +316,36 @@ def test_nurbs_string_types(self):
         # Compare with the reference file
         compare_test_result(self, input_file.get_string(header=False))
 
+    def test_nurbs_hemisphere_surface(self):
+        """Test the creation of the surface of a hemisphere."""
+
+        # Create input file
+        input_file = InputFile()
+
+        # Create the base of a sphere
+        surfs = create_nurbs_hemisphere_surface(2.5, n_ele_uv=3)
```

Comment:
Can we maybe set the elements to`2`? This would reduce the size of the input file, and I think we would still test the general functionality. Or am I missing something?